packages/queries/src/generated/types.test.ts

```diff
@@ -0,0 +1,47 @@
+import { query } from '../query';
+import { Column } from '../types/Column';
+import { ColumnValue } from '../types/ColumnValue';
+import { DB } from '.';
+import { describe, test } from 'vitest';
+import { col } from '../col';
+import { ColumnReference } from '../types/ColumnReference';
+
+describe('New types tests', () => {
```

Comment:
I guess we can remove this file now, all of these cases are covered by existing tests I think